AcctThread.py

```diff
@@ -0,0 +1,280 @@
+#!/usr/local/bin/python3
+
+#    Copyright (c) 2018 Ripple Labs Inc.
+#
+#    Permission to use, copy, modify, and/or distribute this software for any
+#    purpose  with  or without fee is hereby granted, provided that the above
+#    copyright notice and this permission notice appear in all copies.
+#
+#    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+#    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+#    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+
+# Walk back through an account node's history and report all transactions
+# that affect the account in reverse order (from newest to oldest).
+#
+# To capture results redirect stdout to a file.
+
+import json
+import re
+import sys
+import websocket
+
+from websocket import create_connection
+
+# Extract command line arguments.
+def extractArgs ():
```

Comment:
Don't put a space between a function and the argument parens. The standard for formatting python code is pep8. See this section: https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements

I recommend using an automatic formatter for python code. I use yapf (https://github.com/google/yapf) but there are other good formatters out there as well.